### ISSUE TITLE
Fix for menu running out of viewport

### ISSUE THREAD

**Issue by MarcoTroost** [material-components/material-web#5613]
### Description

Hi!

Love the new material components! 
Runs nicely alongside Vue3, without the need for any complicated adapters. 

I've got a slight issue with the menu though. 

A menu with lots of menu-items run of page.
It suggests that the user has to scroll the viewport to view all of the available options (as seen below)

<img width="396" alt="Scherm­afbeelding 2024-05-09 om 09 26 45" src="https://github.com/material-components/material-web/assets/16369103/43cac5a4-fab7-4a34-98f9-6301e4fcb522">

Creating an extra margin between the menu and the border of the viewport would imho help the user experience. This would indicate that the user should scroll the menu instead of the viewport.

<img width="642" alt="Scherm­afbeelding 2024-05-09 om 09 28 01" src="https://github.com/material-components/material-web/assets/16369103/2320c380-614f-47d1-beac-f8a144e6093d">

kind regards,

Marco

**Comment by asyncLiz** [comment]
Good catch! Yes it's supposed to have some default spacing from the edge of the viewport
